Cargo.toml

```diff
@@ -10,7 +10,7 @@ description = "Utility functions for generating Rust code from protobufs (using
 
 [features]
 default = ["protobuf-codec"]
-protobuf-codec = ["protobuf-codegen", "protobuf", "regex"]
+protobuf-codec = ["protobuf-codegen", "protobuf/with-bytes", "regex"]
```

Comment:
Is the feature necessary?

Reply:
Yes, otherwise `Vec<u8>` will still used for bytes.